gnet.go

```diff
@@ -380,15 +389,19 @@ func Run(eventHandler EventHandler, protoAddr string, opts ...Option) (err error
 		options.WriteBufferCap = toolkit.CeilToPowerOfTwo(wbc)
 	}
 
-	network, addr := parseProtoAddr(protoAddr)
+	network, addrs := parseProtoAddr(protoAddr)
 
-	var ln *listener
-	if ln, err = initListener(network, addr, options); err != nil {
-		return
+	listeners := make(map[int]*listener)
+	for _, addr := range strings.Split(addrs, ",") {
```

Comment:
I think we should update parseProtoAddr() to support string format: "tcp://192.168.0.100:8000,tcp://192.168.0.100:8001,tcp://192.168.0.100:8002" to maintain compatibility when gnet supports multiple listener with different protocols.

Reply:
Makes sense, thanks. I will try to fix it and add appropriate tests.